src/OI.cpp

```diff
@@ -61,23 +61,23 @@ OI::OI() {
     dButtonR3.reset(new JoystickButton(driver.get(), 10));
     dButtonR3->WhileHeld(new AutonomousCommand());
     dButtonL3.reset(new JoystickButton(driver.get(), 9));
-    dButtonL3->WhenReleased(new ShifterToggle());
+    dButtonL3->WhenPressed(new ShifterToggle());
```

Comment:
It is true that WhenPressed and WhenRelease are similar but it is quicker
response to the drivers for toggle if we use Whenpressed instead of
WhenReleased

On Mon, Jan 25, 2016 at 6:33 PM, Diego-Andrade <notifications@github.com>
wrote:

> In src/OI.cpp
> <https://github.com/FRC701/RoboVikes2016/pull/8#discussion_r50789563>:
>
> > @@ -61,23 +61,23 @@ OI::OI() {
> >      dButtonR3.reset(new JoystickButton(driver.get(), 10));
> >      dButtonR3->WhileHeld(new AutonomousCommand());
> >      dButtonL3.reset(new JoystickButton(driver.get(), 9));
> > -    dButtonL3->WhenReleased(new ShifterToggle());
> > +    dButtonL3->WhenPressed(new ShifterToggle());
>
> Why the switch? What are the cons and pro of using when pressed to when
> released? Is one better then the other?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/FRC701/RoboVikes2016/pull/8/files#r50789563>.
>